Codigo/FileIO.bas

```diff
@@ -1774,9 +1774,9 @@ Sub CargarBackUp()
 116     frmCargando.cargar.value = 0
 118     frmCargando.ToMapLbl.Visible = True
     
-120     ReDim MapData(1 To NumMaps, XMinMapSize To XMaxMapSize, YMinMapSize To YMaxMapSize) As t_MapBlock
+120     ReDim MapData(1 To (NumMaps + InstanceMapCount), XMinMapSize To XMaxMapSize, YMinMapSize To YMaxMapSize) As t_MapBlock
```

Comment:
Do we really need to load a backup of an instance map ? surely the lifespan is just the even and there should be no need to load a backup?

Reply:
makes sense thanks for the explanation